### DISCUSSION TITLE
Can I use the -id parameter to specify my custom template?

### DISCUSSION THREAD

**Discussion by joner11234** [projectdiscovery/nuclei#5354] [Q&A] [answered]
Can I use the -id parameter to specify my custom template? I tried placing my self-written template in the nuclei-template folder locally and then specifying it with the -id parameter, but it prompted me with "no templates provided for scan".

**Comment by ehsandeep** [comment, marked as answer]
@joner11234 you can as long as you have your custom templates located under same directory as default template installed location.

**Reply by joner11234** [reply]
Thank you, I tried it again and it worked. There was something wrong with my POC writing.
